Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ArchiveWSController.java

```diff
@@ -256,4 +257,22 @@ public ModelAndView unarchiveWorkspace(@Validated @ModelAttribute("workspaceform
 		return model;
 	}
 
+    @AccessPolicies({
+        @ElementAccessPolicy(type = CheckedElementType.PROJECT, paramIndex = 1, userRole = {
+                RoleNames.ROLE_COLLABORATOR_ADMIN,
+                RoleNames.ROLE_PROJ_COLLABORATOR_ADMIN,
+                RoleNames.ROLE_PROJ_COLLABORATOR_CONTRIBUTOR }) })
+    @RequestMapping(value = "auth/workbench/{projectid}/showarchivedworkspace", method = RequestMethod.GET)
+    public String showArchivedWorkspaces(

```

Comment:
1. I did the changes in relation to Show Inactive workspaces which is pointing to location.href='${pageContext.servletContext.contextPath}/auth/workbench/${project.projectid}'" 
2. The method for showing inactive workspace is in ActivateWSController.java file.

But I will make the changes as mentioned